sources/zsh/history.d/haiti

```diff
@@ -0,0 +1,3 @@
+haiti $HASH
+haiti -e $HASH
```

Comment:
Can you use the long option instead (`--extended`) so that it's self explanatory?